src/components/commandHandler.ts

```diff
@@ -27,114 +26,114 @@ interface SupportedCommands {
     [command: string]: (data: DataMessage) => any;
 }
 
-export class commandHandler {
```

Comment:
This class didn't need instantiation, we were just keeping one instance in window.commandHandler before.